source/features/update-pr-from-master.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import * as api from '../libs/api';
+import * as icons from '../libs/icons';
+
+export async function mergeBranches(repoUrl: string, base: string, head: string) {
+	const result = await api.v3(`repos/${repoUrl}/merges`, {
```

Comment:
The `v4` API doesn't support this?

Reply:
I think not, and even then the REST query is easier to build